src/styles/blocks/body.scss

```
@@ -0,0 +1,4 @@
+.body {
```

and name this file as `common.scss`
```suggestion
body {
```